glrd/manage.py

```diff
@@ -1225,25 +1310,35 @@ def parse_arguments():
     parser.add_argument('--no-query', action='store_true', help="Do not query and use existing releases using glrd command. Be careful, this can delete your releases.")
     parser.add_argument('--input-stdin', action='store_true', help="Process a single input from stdin (JSON data).")
     parser.add_argument('--input', action='store_true', help="Process input from --input-file.")
-    parser.add_argument('--input-file', type=str, default=DEFAULTS['DEFAULT_MANAGE_INPUT_FILE'], help="The name of the input file (default: releases-input.yaml).")
-    parser.add_argument('--output-file-prefix', type=str, default=DEFAULTS['DEFAULT_MANAGE_OUTPUT_FILE_PREFIX'], help="The prefix added to the output file (default: releases).")
-    parser.add_argument('--output-format', type=str, choices=['yaml', 'json'], default=DEFAULTS['DEFAULT_MANAGE_OUTPUT_FORMAT'], help="Output format: 'yaml' or 'json' (default: json).")
     parser.add_argument('--no-output-split', action='store_true', help="Do not split Output into stable+patch and nightly. Additional output-files *-nightly and *-dev will not be created.")
-    parser.add_argument('--s3-bucket-name', type=str, default=DEFAULTS['DEFAULT_S3_BUCKET_NAME'], help="Name of S3 bucket. Defaults to 'gardenlinux-glrd'.")
-    parser.add_argument('--s3-bucket-prefix', type=str, default=DEFAULTS['DEFAULT_S3_BUCKET_PREFIX'], help="Prefix inside S3 bucket. Defaults to ''.")
-    parser.add_argument('--s3-bucket-region', type=str, default=DEFAULTS['DEFAULT_S3_BUCKET_REGION'], help="Name of S3 bucket Region. Defaults to 'eu-central-1'.")
     parser.add_argument('--s3-create-bucket', action='store_true', help="Create an S3 bucket.")
     parser.add_argument('--s3-update', action='store_true', help="Update (merge) the generated files with S3.")
-    parser.add_argument('--log-level', type=str, choices=['ERROR', 'WARNING', 'INFO', 'DEBUG'], default='INFO', help="Set the logging level (default: INFO).")
     parser.add_argument('--output-all', action='store_true', 
                        help="Download and write all release files found in S3 to local disk")
     parser.add_argument('--input-all', action='store_true',
                        help="Upload all local release files to S3")
 
     args = parser.parse_args()
 
+    # Convert log level to uppercase if provided in lowercase
+    args.log_level = args.log_level.upper()
+
     if len(sys.argv) == 1:
         parser.print_help()
         sys.exit(ERROR_CODES["parameter_missing"])
 
     return args
+
+def main():
+    args = parse_arguments()
+
+    # Configure logging with the already uppercase level
+    logging.basicConfig(
+        level=args.log_level,
+        format='%(levelname)s: %(message)s'
+    )
+
+    handle_releases(args)
+
+if __name__ == "__main__":
+    main()
```

Comment:
With 1344 lines this file is way too large. We should refactor and reuse code.

Reply:
https://github.com/gardenlinux/glrd/issues/4